addons/weave/template/base/patch-daemonset.yaml

```diff
@@ -22,3 +22,12 @@ spec:
               value: "weaveworks/weaveexec:__WEAVE_VERSION__"
```

Comment:
@emosbaugh I tested it out and seems that it worked.

Following the steps and the result
1 - Re-generate the weave addon latest release by removing it and calling the generate.sh script
2 - I used the following KURL install

```yaml
INSTALLER_YAML="apiVersion: cluster.kurl.sh/v1beta1
kind: Installer
metadata:
  name: latest
spec:
  kubernetes:
    version: 1.23.17
  weave:
    version: 2.8.1-20230417
  openebs:
    version: 3.3.0
    isLocalPVEnabled: true
    localPVStorageClassName: default
  minio:
    version: 2022-10-20T00-55-09Z
  contour:
    version: 1.23.0
  registry:
    version: 2.8.1
  prometheus:
    version: 0.60.1-41.7.3
  kurl:
    installerVersion: v2023.05.15-0
    additionalNoProxyAddresses: []
  containerd:
    version: 1.6.8"
``` 
3 - I created the following symbolic links

```sh
$ sudo bash ./test.sh 
+ FOLDER=/securiti_ai_tenant
+ mkdir -p /securiti_ai_tenant/opt/cni
+ mkdir -p /securiti_ai_tenant/opt/containerd
+ mkdir -p /securiti_ai_tenant/opt/ekco
+ mkdir -p /securiti_ai_tenant/opt/replicated
+ mkdir -p /securiti_ai_tenant/mnt/drivers
+ mkdir -p /securiti_ai_tenant/mnt/tmpfs
+ mkdir -p /securiti_ai_tenant/var/lib/cni
+ mkdir -p /securiti_ai_tenant/var/lib/containerd
+ mkdir -p /securiti_ai_tenant/var/lib/containers
+ mkdir -p /securiti_ai_tenant/var/lib/docker
+ mkdir -p /securiti_ai_tenant/var/lib/dockershim
+ mkdir -p /securiti_ai_tenant/var/lib/etcd
+ mkdir -p /securiti_ai_tenant/var/lib/kubelet
+ mkdir -p /securiti_ai_tenant/var/lib/kurl
+ mkdir -p /securiti_ai_tenant/var/lib/rook
+ mkdir -p /securiti_ai_tenant/var/lib/weave
+ ln -s /securiti_ai_tenant/opt/cni /opt/cni
+ ln -s /securiti_ai_tenant/opt/containerd /opt/containerd
+ ln -s /securiti_ai_tenant/opt/ekco /opt/ekco
+ ln -s /securiti_ai_tenant/opt/replicated /opt/replicated
+ ln -s /securiti_ai_tenant/mnt/drivers /mnt/drivers
+ ln -s /securiti_ai_tenant/mnt/tmpfs /mnt/tmpfs
+ ln -s /securiti_ai_tenant/var/lib/cni /var/lib/cni
+ ln -s /securiti_ai_tenant/var/lib/containerd /var/lib/containerd
+ ln -s /securiti_ai_tenant/var/lib/containers /var/lib/containers
+ ln -s /securiti_ai_tenant/var/lib/docker /var/lib/docker
+ ln -s /securiti_ai_tenant/var/lib/dockershim /var/lib/dockershim
+ ln -s /securiti_ai_tenant/var/lib/etcd /var/lib/etcd
+ ln -s /securiti_ai_tenant/var/lib/kubelet /var/lib/kubelet
+ ln -s /securiti_ai_tenant/var/lib/kurl /var/lib/kurl
+ ln -s /securiti_ai_tenant/var/lib/rook /var/lib/rook
+ ln -s /securiti_ai_tenant/var/lib/weave /var/lib/weave
```

4 - Following the logs of its install into the created folder


```sh
 cat install-2023-05-16T15-21-19.log 
⚙  Running install with the argument(s): 
Failed to get unit file state for firewalld.service: No such file or directory
✔ Firewalld is either not enabled or not active.
SELinux is not installed: no configuration will be applied
The installer will use network interface 'ens4' (with IP address '10.154.0.61')
⚙  Running host preflights
[TCP Port Status] Running collector...
[CPU Info] Running collector...
[Amount of Memory] Running collector...
[Block Devices] Running collector...
[Host OS Info] Running collector...
[Ephemeral Disk Usage /var/lib/kubelet] Running collector...
[Ephemeral Disk Usage /var/lib/containerd] Running collector...
[Ephemeral Disk Usage /var/openebs] Running collector...
[Kubernetes API TCP Port Status] Running collector...
[ETCD Client API TCP Port Status] Running collector...
[ETCD Server API TCP Port Status] Running collector...
[ETCD Health Server TCP Port Status] Running collector...
[Kubelet Health Server TCP Port Status] Running collector...
[Kubelet API TCP Port Status] Running collector...
[Kube Controller Manager Health Server TCP Port Status] Running collector...
[Kube Scheduler Health Server TCP Port Status] Running collector...
[Filesystem Latency Two Minute Benchmark] Running collector...
[Host OS Info] Running collector...
[Weave Network Policy Controller Metrics Server TCP Port Status] Running collector...
[Weave Net Metrics Server TCP Port Status] Running collector...
[Weave Net Control TCP Port Status] Running collector...
[Node Exporter Metrics Server TCP Port Status] Running collector...
[Host OS Info] Running collector...
I0516 15:22:18.007775    6546 analyzer.go:77] excluding "Certificate Key Pair" analyzer
I0516 15:22:18.007795    6546 analyzer.go:77] excluding "Certificate Key Pair" analyzer
I0516 15:22:18.007802    6546 analyzer.go:77] excluding "Kubernetes API Health" analyzer
I0516 15:22:18.007871    6546 analyzer.go:77] excluding "Ephemeral Disk Usage /var/lib/docker" analyzer
I0516 15:22:18.007893    6546 analyzer.go:77] excluding "Ephemeral Disk Usage /var/lib/rook" analyzer
I0516 15:22:18.007912    6546 analyzer.go:77] excluding "Kubernetes API Server Load Balancer" analyzer
I0516 15:22:18.007918    6546 analyzer.go:77] excluding "Kubernetes API Server Load Balancer Upgrade" analyzer
I0516 15:22:18.007967    6546 analyzer.go:77] excluding "Kubernetes API TCP Connection Status" analyzer
I0516 15:22:18.008075    6546 analyzer.go:77] excluding "Docker Support" analyzer
I0516 15:22:18.008087    6546 analyzer.go:77] excluding "Containerd and Weave Compatibility" analyzer
[PASS] Number of CPUs: This server has at least 4 CPU cores
[PASS] Amount of Memory: The system has at least 8G of memory
[PASS] Ephemeral Disk Usage /var/lib/kubelet: The disk containing directory /var/lib/kubelet has at least 30Gi of total space, has at least 10Gi of disk space available, and is less than 60% full
[PASS] Ephemeral Disk Usage /var/lib/containerd: The disk containing directory /var/lib/containerd has at least 30Gi of total space, has at least 10Gi of disk space available, and is less than 60% full.
[PASS] Ephemeral Disk Usage /var/openebs: The disk containing directory /var/openebs has sufficient space
[PASS] Kubernetes API TCP Port Status: Port 6443 is open
[PASS] ETCD Client API TCP Port Status: Port 2379 is open
[PASS] ETCD Server API TCP Port Status: Port 2380 is open
[PASS] ETCD Health Server TCP Port Status: Port 2381 is available
[PASS] Kubelet Health Server TCP Port Status: Port 10248 is available
[PASS] Kubelet API TCP Port Status: Port 10250 is open
[PASS] Kube Controller Manager Health Server TCP Port Status: Port 10257 is available
[PASS] Kube Scheduler Health Server TCP Port Status: Port 10259 is available
[PASS] Filesystem Performance: Write latency is ok (p99 target < 10ms, actual: 3.050241ms)
[PASS] NTP Status: System clock is synchronized
[PASS] NTP Status: Timezone is set to UTC
[PASS] Host OS Info: containerd addon supports ubuntu 22.04
[PASS] Weave Network Policy Controller Metrics Server TCP Port Status: Port 6781 is available
[PASS] Weave Net Metrics Server TCP Port Status: Port 6782 is available
[PASS] Weave Net Control TCP Port Status: Port 6783 is open
[PASS] Node Exporter Metrics Server TCP Port Status: Port 9100 is available
[PASS] Host OS Info: containerd addon supports ubuntu 22.04
⚙  Host preflights success
Found pod network: 10.32.0.0/20
Found service network: 10.96.0.0/22
⚙  Addon containerd 1.6.8
⚙  Installing host packages containerd.io
Selecting previously unselected package containerd.io.
(Reading database ... 64969 files and directories currently installed.)
Preparing to unpack .../containerd.io_1.6.8-1_amd64.deb ...
Unpacking containerd.io (1.6.8-1) ...
Setting up containerd.io (1.6.8-1) ...
Created symlink /etc/systemd/system/multi-user.target.wants/containerd.service → /lib/systemd/system/containerd.service.
Processing triggers for man-db (2.10.2-1) ...
✔ Host packages containerd.io installed
⚙  Checking package manager status
Reading package lists...
Building dependency tree...
Reading state information...
✔ Status checked successfully. No broken packages were found.
Restarting containerd...
Service containerd restarted.
unpacking k8s.gcr.io/pause:3.6 (sha256:79b611631c0d19e9a975fb0a8511e5153789b4c26610d1842e9f735c57cc8b13)...done
registry.k8s.io/pause:3.6
Adding kernel module br_netfilter
* Applying /etc/sysctl.d/10-console-messages.conf ...
kernel.printk = 4 4 1 7
* Applying /etc/sysctl.d/10-ipv6-privacy.conf ...
net.ipv6.conf.all.use_tempaddr = 2
net.ipv6.conf.default.use_tempaddr = 2
* Applying /etc/sysctl.d/10-kernel-hardening.conf ...
kernel.kptr_restrict = 1
* Applying /etc/sysctl.d/10-magic-sysrq.conf ...
kernel.sysrq = 176
* Applying /etc/sysctl.d/10-network-security.conf ...
net.ipv4.conf.default.rp_filter = 2
net.ipv4.conf.all.rp_filter = 2
* Applying /etc/sysctl.d/10-ptrace.conf ...
kernel.yama.ptrace_scope = 1
* Applying /etc/sysctl.d/10-zeropage.conf ...
vm.mmap_min_addr = 65536
* Applying /usr/lib/sysctl.d/50-default.conf ...
kernel.core_uses_pid = 1
net.ipv4.conf.default.rp_filter = 2
net.ipv4.conf.default.accept_source_route = 0
sysctl: setting key "net.ipv4.conf.all.accept_source_route": Invalid argument
net.ipv4.conf.default.promote_secondaries = 1
sysctl: setting key "net.ipv4.conf.all.promote_secondaries": Invalid argument
net.ipv4.ping_group_range = 0 2147483647
net.core.default_qdisc = fq_codel
fs.protected_hardlinks = 1
fs.protected_symlinks = 1
fs.protected_regular = 1
fs.protected_fifos = 1
* Applying /usr/lib/sysctl.d/50-pid-max.conf ...
kernel.pid_max = 4194304
* Applying /etc/sysctl.d/60-gce-network-security.conf ...
net.ipv4.tcp_syncookies = 1
net.ipv4.conf.all.accept_source_route = 0
net.ipv4.conf.default.accept_source_route = 0
net.ipv4.conf.all.accept_redirects = 0
net.ipv4.conf.default.accept_redirects = 0
net.ipv4.conf.all.secure_redirects = 1
net.ipv4.conf.default.secure_redirects = 1
net.ipv4.ip_forward = 0
net.ipv4.conf.all.send_redirects = 0
net.ipv4.conf.default.send_redirects = 0
net.ipv4.conf.all.rp_filter = 1
net.ipv4.conf.default.rp_filter = 1
net.ipv4.icmp_echo_ignore_broadcasts = 1
net.ipv4.icmp_ignore_bogus_error_responses = 1
net.ipv4.conf.all.log_martians = 1
net.ipv4.conf.default.log_martians = 1
kernel.randomize_va_space = 2
kernel.panic = 10
* Applying /etc/sysctl.d/99-cloudimg-ipv6.conf ...
net.ipv6.conf.all.use_tempaddr = 0
net.ipv6.conf.default.use_tempaddr = 0
* Applying /usr/lib/sysctl.d/99-protect-links.conf ...
fs.protected_fifos = 1
fs.protected_hardlinks = 1
fs.protected_regular = 2
fs.protected_symlinks = 1
* Applying /etc/sysctl.d/99-replicated-ipv4.conf ...
net.bridge.bridge-nf-call-iptables = 1
net.ipv4.conf.all.forwarding = 1
* Applying /etc/sysctl.d/99-sysctl.conf ...
* Applying /etc/sysctl.conf ...
Adding kernel modules ip_vs, ip_vs_rr, ip_vs_wrr, and ip_vs_sh
⚙  Install kubelet, kubectl and cni host packages
kubelet command missing - will install host components
⚙  Installing host packages kubelet-1.23.17 kubectl-1.23.17 git
Selecting previously unselected package conntrack.
(Reading database ... 64985 files and directories currently installed.)
Preparing to unpack .../conntrack_1%3a1.4.6-2build2_amd64.deb ...
Unpacking conntrack (1:1.4.6-2build2) ...
Selecting previously unselected package ebtables.
Preparing to unpack .../ebtables_2.0.11-4build2_amd64.deb ...
Unpacking ebtables (2.0.11-4build2) ...
Preparing to unpack .../ethtool_1%3a5.16-1_amd64.deb ...
Unpacking ethtool (1:5.16-1) over (1:5.16-1) ...
Preparing to unpack .../git-man_1%3a2.34.1-1ubuntu1.9_all.deb ...
Unpacking git-man (1:2.34.1-1ubuntu1.9) over (1:2.34.1-1ubuntu1.8) ...
Preparing to unpack .../git_1%3a2.34.1-1ubuntu1.9_amd64.deb ...
Unpacking git (1:2.34.1-1ubuntu1.9) over (1:2.34.1-1ubuntu1.8) ...
Preparing to unpack .../iproute2_5.15.0-1ubuntu2_amd64.deb ...
Unpacking iproute2 (5.15.0-1ubuntu2) over (5.15.0-1ubuntu2) ...
Preparing to unpack .../iptables_1.8.7-1ubuntu5_amd64.deb ...
Unpacking iptables (1.8.7-1ubuntu5) over (1.8.7-1ubuntu5) ...
Preparing to unpack .../kmod_29-1ubuntu1_amd64.deb ...
Unpacking kmod (29-1ubuntu1) over (29-1ubuntu1) ...
Selecting previously unselected package kubectl.
Preparing to unpack .../kubectl_1.23.17-00_amd64.deb ...
Unpacking kubectl (1.23.17-00) ...
Selecting previously unselected package kubelet.
Preparing to unpack .../kubelet_1.23.17-00_amd64.deb ...
Unpacking kubelet (1.23.17-00) ...
Selecting previously unselected package kubernetes-cni.
Preparing to unpack .../kubernetes-cni_1.2.0-00_amd64.deb ...
Unpacking kubernetes-cni (1.2.0-00) ...
Preparing to unpack .../less_590-1ubuntu0.22.04.1_amd64.deb ...
Unpacking less (590-1ubuntu0.22.04.1) over (590-1ubuntu0.22.04.1) ...
Preparing to unpack .../libatm1_1%3a2.5.1-4build2_amd64.deb ...
Unpacking libatm1:amd64 (1:2.5.1-4build2) over (1:2.5.1-4build2) ...
Preparing to unpack .../libbpf0_1%3a0.5.0-1ubuntu22.04.1_amd64.deb ...
Unpacking libbpf0:amd64 (1:0.5.0-1ubuntu22.04.1) over (1:0.5.0-1ubuntu22.04.1) ...
Preparing to unpack .../libbsd0_0.11.5-1_amd64.deb ...
Unpacking libbsd0:amd64 (0.11.5-1) over (0.11.5-1) ...
Preparing to unpack .../libcap2-bin_1%3a2.44-1build3_amd64.deb ...
Unpacking libcap2-bin (1:2.44-1build3) over (1:2.44-1build3) ...
Preparing to unpack .../libcbor0.8_0.8.0-2ubuntu1_amd64.deb ...
Unpacking libcbor0.8:amd64 (0.8.0-2ubuntu1) over (0.8.0-2ubuntu1) ...
Preparing to unpack .../libcurl3-gnutls_7.81.0-1ubuntu1.10_amd64.deb ...
Unpacking libcurl3-gnutls:amd64 (7.81.0-1ubuntu1.10) over (7.81.0-1ubuntu1.10) ...
Preparing to unpack .../libedit2_3.1-20210910-1build1_amd64.deb ...
Unpacking libedit2:amd64 (3.1-20210910-1build1) over (3.1-20210910-1build1) ...
Preparing to unpack .../libelf1_0.186-1build1_amd64.deb ...
Unpacking libelf1:amd64 (0.186-1build1) over (0.186-1build1) ...
Preparing to unpack .../liberror-perl_0.17029-1_all.deb ...
Unpacking liberror-perl (0.17029-1) over (0.17029-1) ...
Preparing to unpack .../libexpat1_2.4.7-1ubuntu0.2_amd64.deb ...
Unpacking libexpat1:amd64 (2.4.7-1ubuntu0.2) over (2.4.7-1ubuntu0.2) ...
Preparing to unpack .../libfido2-1_1.10.0-1_amd64.deb ...
Unpacking libfido2-1:amd64 (1.10.0-1) over (1.10.0-1) ...
Preparing to unpack .../libgdbm-compat4_1.23-1_amd64.deb ...
Unpacking libgdbm-compat4:amd64 (1.23-1) over (1.23-1) ...
Preparing to unpack .../libgdbm6_1.23-1_amd64.deb ...
Unpacking libgdbm6:amd64 (1.23-1) over (1.23-1) ...
Preparing to unpack .../libip4tc2_1.8.7-1ubuntu5_amd64.deb ...
Unpacking libip4tc2:amd64 (1.8.7-1ubuntu5) over (1.8.7-1ubuntu5) ...
Preparing to unpack .../libip6tc2_1.8.7-1ubuntu5_amd64.deb ...
Unpacking libip6tc2:amd64 (1.8.7-1ubuntu5) over (1.8.7-1ubuntu5) ...
Preparing to unpack .../libkmod2_29-1ubuntu1_amd64.deb ...
Unpacking libkmod2:amd64 (29-1ubuntu1) over (29-1ubuntu1) ...
Preparing to unpack .../libmd0_1.0.4-1build1_amd64.deb ...
Unpacking libmd0:amd64 (1.0.4-1build1) over (1.0.4-1build1) ...
Preparing to unpack .../libmnl0_1.0.4-3build2_amd64.deb ...
Unpacking libmnl0:amd64 (1.0.4-3build2) over (1.0.4-3build2) ...
Preparing to unpack .../libnetfilter-conntrack3_1.0.9-1_amd64.deb ...
Unpacking libnetfilter-conntrack3:amd64 (1.0.9-1) over (1.0.9-1) ...
Preparing to unpack .../libnfnetlink0_1.0.1-3build3_amd64.deb ...
Unpacking libnfnetlink0:amd64 (1.0.1-3build3) over (1.0.1-3build3) ...
Preparing to unpack .../libnftnl11_1.2.1-1build1_amd64.deb ...
Unpacking libnftnl11:amd64 (1.2.1-1build1) over (1.2.1-1build1) ...
Preparing to unpack .../libpam-cap_1%3a2.44-1build3_amd64.deb ...
Unpacking libpam-cap:amd64 (1:2.44-1build3) over (1:2.44-1build3) ...
Preparing to unpack .../libperl5.34_5.34.0-3ubuntu1.1_amd64.deb ...
Unpacking libperl5.34:amd64 (5.34.0-3ubuntu1.1) over (5.34.0-3ubuntu1.1) ...
Preparing to unpack .../libwrap0_7.6.q-31build2_amd64.deb ...
Unpacking libwrap0:amd64 (7.6.q-31build2) over (7.6.q-31build2) ...
Preparing to unpack .../libx11-6_2%3a1.7.5-1_amd64.deb ...
Unpacking libx11-6:amd64 (2:1.7.5-1) over (2:1.7.5-1) ...
Preparing to unpack .../libx11-data_2%3a1.7.5-1_all.deb ...
Unpacking libx11-data (2:1.7.5-1) over (2:1.7.5-1) ...
Preparing to unpack .../libxau6_1%3a1.0.9-1build5_amd64.deb ...
Unpacking libxau6:amd64 (1:1.0.9-1build5) over (1:1.0.9-1build5) ...
Preparing to unpack .../libxcb1_1.14-3ubuntu3_amd64.deb ...
Unpacking libxcb1:amd64 (1.14-3ubuntu3) over (1.14-3ubuntu3) ...
Preparing to unpack .../libxdmcp6_1%3a1.1.3-0ubuntu5_amd64.deb ...
Unpacking libxdmcp6:amd64 (1:1.1.3-0ubuntu5) over (1:1.1.3-0ubuntu5) ...
Preparing to unpack .../libxext6_2%3a1.3.4-1build1_amd64.deb ...
Unpacking libxext6:amd64 (2:1.3.4-1build1) over (2:1.3.4-1build1) ...
Preparing to unpack .../libxmuu1_2%3a1.1.3-3_amd64.deb ...
Unpacking libxmuu1:amd64 (2:1.1.3-3) over (2:1.1.3-3) ...
Preparing to unpack .../libxtables12_1.8.7-1ubuntu5_amd64.deb ...
Unpacking libxtables12:amd64 (1.8.7-1ubuntu5) over (1.8.7-1ubuntu5) ...
Preparing to unpack .../netbase_6.3_all.deb ...
Unpacking netbase (6.3) over (6.3) ...
Preparing to unpack .../openssh-client_1%3a8.9p1-3ubuntu0.1_amd64.deb ...
Unpacking openssh-client (1:8.9p1-3ubuntu0.1) over (1:8.9p1-3ubuntu0.1) ...
Preparing to unpack .../patch_2.7.6-7build2_amd64.deb ...
Unpacking patch (2.7.6-7build2) over (2.7.6-7build2) ...
Preparing to unpack .../perl-modules-5.34_5.34.0-3ubuntu1.1_all.deb ...
Unpacking perl-modules-5.34 (5.34.0-3ubuntu1.1) over (5.34.0-3ubuntu1.1) ...
Preparing to unpack .../perl_5.34.0-3ubuntu1.1_amd64.deb ...
Unpacking perl (5.34.0-3ubuntu1.1) over (5.34.0-3ubuntu1.1) ...
Selecting previously unselected package socat.
Preparing to unpack .../socat_1.7.4.1-3ubuntu4_amd64.deb ...
Unpacking socat (1.7.4.1-3ubuntu4) ...
Preparing to unpack .../xauth_1%3a1.1-1build2_amd64.deb ...
Unpacking xauth (1:1.1-1build2) over (1:1.1-1build2) ...
Setting up git-man (1:2.34.1-1ubuntu1.9) ...
Setting up kubectl (1.23.17-00) ...
Setting up kubernetes-cni (1.2.0-00) ...
Setting up less (590-1ubuntu0.22.04.1) ...
Setting up libatm1:amd64 (1:2.5.1-4build2) ...
Setting up libcap2-bin (1:2.44-1build3) ...
Setting up libcbor0.8:amd64 (0.8.0-2ubuntu1) ...
Setting up libcurl3-gnutls:amd64 (7.81.0-1ubuntu1.10) ...
Setting up libelf1:amd64 (0.186-1build1) ...
Setting up libexpat1:amd64 (2.4.7-1ubuntu0.2) ...
Setting up libfido2-1:amd64 (1.10.0-1) ...
Setting up libgdbm6:amd64 (1.23-1) ...
Setting up libip4tc2:amd64 (1.8.7-1ubuntu5) ...
Setting up libip6tc2:amd64 (1.8.7-1ubuntu5) ...
Setting up libkmod2:amd64 (29-1ubuntu1) ...
Setting up libmd0:amd64 (1.0.4-1build1) ...
Setting up libmnl0:amd64 (1.0.4-3build2) ...
Setting up libnfnetlink0:amd64 (1.0.1-3build3) ...
Setting up libnftnl11:amd64 (1.2.1-1build1) ...
Setting up libpam-cap:amd64 (1:2.44-1build3) ...
Setting up libwrap0:amd64 (7.6.q-31build2) ...
Setting up libx11-data (2:1.7.5-1) ...
Setting up libxau6:amd64 (1:1.0.9-1build5) ...
Setting up libxtables12:amd64 (1.8.7-1ubuntu5) ...
Setting up netbase (6.3) ...
Setting up patch (2.7.6-7build2) ...
Setting up perl-modules-5.34 (5.34.0-3ubuntu1.1) ...
Setting up socat (1.7.4.1-3ubuntu4) ...
Setting up ebtables (2.0.11-4build2) ...
Setting up ethtool (1:5.16-1) ...
Setting up kmod (29-1ubuntu1) ...
Setting up libbpf0:amd64 (1:0.5.0-1ubuntu22.04.1) ...
Setting up libbsd0:amd64 (0.11.5-1) ...
Setting up libedit2:amd64 (3.1-20210910-1build1) ...
Setting up libgdbm-compat4:amd64 (1.23-1) ...
Setting up libnetfilter-conntrack3:amd64 (1.0.9-1) ...
Setting up libperl5.34:amd64 (5.34.0-3ubuntu1.1) ...
Setting up libxdmcp6:amd64 (1:1.1.3-0ubuntu5) ...
Setting up openssh-client (1:8.9p1-3ubuntu0.1) ...
Setting up perl (5.34.0-3ubuntu1.1) ...
Setting up conntrack (1:1.4.6-2build2) ...
Setting up iproute2 (5.15.0-1ubuntu2) ...
Setting up iptables (1.8.7-1ubuntu5) ...
Setting up kubelet (1.23.17-00) ...
Created symlink /etc/systemd/system/multi-user.target.wants/kubelet.service → /lib/systemd/system/kubelet.service.
Setting up liberror-perl (0.17029-1) ...
Setting up libxcb1:amd64 (1.14-3ubuntu3) ...
Setting up git (1:2.34.1-1ubuntu1.9) ...
Setting up libx11-6:amd64 (2:1.7.5-1) ...
Setting up libxext6:amd64 (2:1.3.4-1build1) ...
Setting up libxmuu1:amd64 (2:1.1.3-3) ...
Setting up xauth (1:1.1-1build2) ...
Processing triggers for man-db (2.10.2-1) ...
Processing triggers for initramfs-tools (0.140ubuntu13.1) ...
update-initramfs: Generating /boot/initrd.img-5.19.0-1022-gcp
Processing triggers for libc-bin (2.35-0ubuntu3.1) ...
✔ Host packages kubelet-1.23.17 kubectl-1.23.17 git installed
⚙  Checking package manager status
Reading package lists...
Building dependency tree...
Reading state information...
✔ Status checked successfully. No broken packages were found.
Restarting Kubelet
✔ Kubernetes host packages installed
unpacking registry.k8s.io/kube-controller-manager:v1.23.17 (sha256:af4668e3241683d1aa2927c7e59afe1733b5c969582d210fde66bc30cb47097f)...done
unpacking registry.k8s.io/coredns/coredns:v1.8.6 (sha256:7aba92733295a77c554e143d7a625c5cad620944b5bce174faeb512f9c61c277)...done
unpacking registry.k8s.io/kube-apiserver:v1.23.17 (sha256:0bb360812249ee15ed49d42f41aaee97abb1f61ccf72e48bd8e1269925de555b)...done
unpacking registry.k8s.io/pause:3.6 (sha256:79b611631c0d19e9a975fb0a8511e5153789b4c26610d1842e9f735c57cc8b13)...done
unpacking registry.k8s.io/etcd:3.5.6-0 (sha256:00e797072c1d464279130edbd58cbe862ff94972b82aeac5c0786b6278e21455)...done
unpacking registry.k8s.io/kube-scheduler:v1.23.17 (sha256:2675bddfcd0a6c3a614e69683728760c0833c3a16089ee8de2f3c211abac3408)...done
unpacking registry.k8s.io/kube-proxy:v1.23.17 (sha256:25fad4282ccc51dc5ab4940748e2f1d8316a81c04454321ad484fe8528fff2ea)...done
/securiti_ai_tenant/kurl/krew /securiti_ai_tenant/kurl /securiti_ai_tenant
/securiti_ai_tenant/kurl /securiti_ai_tenant
/securiti_ai_tenant/kurl/packages/kubernetes/1.23.17/assets /securiti_ai_tenant/kurl /securiti_ai_tenant
/securiti_ai_tenant/kurl /securiti_ai_tenant
unpacking k8s.gcr.io/pause:3.6 (sha256:79b611631c0d19e9a975fb0a8511e5153789b4c26610d1842e9f735c57cc8b13)...done
unpacking docker.io/kurlsh/weave-npc:2.8.1-20230417-4a9fe55 (sha256:e975dd8d8aa3d5bf56e0140fad38a917ce1571c1ce4e1436be8a6b39a4756231)...done
unpacking docker.io/kurlsh/weaveexec:2.8.1-20230417-4a9fe55 (sha256:6ea03c3c2fdd5d6554809ab0840641429cdde8b3f86cbdaf4de50734b814383c)...done
unpacking docker.io/kurlsh/weave-kube:2.8.1-20230417-4a9fe55 (sha256:dc9c5d63b57f88cb550fdd7a5c990672c937eb2f78f0d501c74eaf7793a965df)...done
unpacking docker.io/openebs/linux-utils:3.3.0 (sha256:31463b410f1660cd670a71bb529abec86889c0b860cb12d7aa04f63db6e97b12)...done
unpacking docker.io/openebs/node-disk-manager:2.0.0 (sha256:54c123bf69205d7de97d04256f0584867ef739ef312888648a4fcb7361804ff8)...done
unpacking docker.io/openebs/provisioner-localpv:3.3.0 (sha256:6b89119d935697b8ef2de91c4158734b45a7d4a32c07bd18a25caca1570b85f3)...done
unpacking docker.io/openebs/node-disk-operator:2.0.0 (sha256:199ca906372795292aac828818bd9f7c262db8eeb72d391dc4bb763b1076b48d)...done
find: ‘./addons/minio/2022-10-20T00-55-09Z/images’: No such file or directory
find: ‘./addons/contour/1.23.0/images’: No such file or directory
find: ‘./addons/registry/2.8.1/images’: No such file or directory
find: ‘./addons/prometheus/0.60.1-41.7.3/images’: No such file or directory
⚙  Initialize Kubernetes
⚙  generate kubernetes bootstrap token
Kubernetes bootstrap token: i6skv0.j53bfnee6vqvsv10
This token will expire in 24 hours
[init] Using Kubernetes version: v1.23.17
[preflight] Running pre-flight checks
	[WARNING DirAvailable--var-lib-etcd]: /var/lib/etcd is not empty
[preflight] Pulling images required for setting up a Kubernetes cluster
[preflight] This might take a minute or two, depending on the speed of your internet connection
[preflight] You can also perform this action in beforehand using 'kubeadm config images pull'
[certs] Using certificateDir folder "/etc/kubernetes/pki"
[certs] Generating "ca" certificate and key
[certs] Generating "apiserver" certificate and key
[certs] apiserver serving cert is signed for DNS names [camila-ubuntu-2204-test-pr-weave-2 kubernetes kubernetes.default kubernetes.default.svc kubernetes.default.svc.cluster.local] and IPs [10.96.0.1 10.154.0.61 34.105.233.9]
[certs] Generating "apiserver-kubelet-client" certificate and key
[certs] Generating "front-proxy-ca" certificate and key
[certs] Generating "front-proxy-client" certificate and key
[certs] Generating "etcd/ca" certificate and key
[certs] Generating "etcd/server" certificate and key
[certs] etcd/server serving cert is signed for DNS names [camila-ubuntu-2204-test-pr-weave-2 localhost] and IPs [10.154.0.61 127.0.0.1 ::1]
[certs] Generating "etcd/peer" certificate and key
[certs] etcd/peer serving cert is signed for DNS names [camila-ubuntu-2204-test-pr-weave-2 localhost] and IPs [10.154.0.61 127.0.0.1 ::1]
[certs] Generating "etcd/healthcheck-client" certificate and key
[certs] Generating "apiserver-etcd-client" certificate and key
[certs] Generating "sa" key and public key
[kubeconfig] Using kubeconfig folder "/etc/kubernetes"
[kubeconfig] Writing "admin.conf" kubeconfig file
[kubeconfig] Writing "kubelet.conf" kubeconfig file
[kubeconfig] Writing "controller-manager.conf" kubeconfig file
[kubeconfig] Writing "scheduler.conf" kubeconfig file
[kubelet-start] Writing kubelet environment file with flags to file "/var/lib/kubelet/kubeadm-flags.env"
[kubelet-start] Writing kubelet configuration to file "/var/lib/kubelet/config.yaml"
[kubelet-start] Starting the kubelet
[control-plane] Using manifest folder "/etc/kubernetes/manifests"
[control-plane] Creating static Pod manifest for "kube-apiserver"
[control-plane] Creating static Pod manifest for "kube-controller-manager"
[control-plane] Creating static Pod manifest for "kube-scheduler"
[etcd] Creating static Pod manifest for local etcd in "/etc/kubernetes/manifests"
[wait-control-plane] Waiting for the kubelet to boot up the control plane as static Pods from directory "/etc/kubernetes/manifests". This can take up to 4m0s
[apiclient] All control plane components are healthy after 11.016931 seconds
[upload-config] Storing the configuration used in ConfigMap "kubeadm-config" in the "kube-system" Namespace
[kubelet] Creating a ConfigMap "kubelet-config-1.23" in namespace kube-system with the configuration for the kubelets in the cluster
NOTE: The "kubelet-config-1.23" naming of the kubelet ConfigMap is deprecated. Once the UnversionedKubeletConfigMap feature gate graduates to Beta the default name will become just "kubelet-config". Kubeadm upgrade will handle this transition transparently.
[upload-certs] Skipping phase. Please see --upload-certs
[mark-control-plane] Marking the node camila-ubuntu-2204-test-pr-weave-2 as control-plane by adding the labels: [node-role.kubernetes.io/master(deprecated) node-role.kubernetes.io/control-plane node.kubernetes.io/exclude-from-external-load-balancers]
[bootstrap-token] Using token: i6skv0.j53bfnee6vqvsv10
[bootstrap-token] Configuring bootstrap tokens, cluster-info ConfigMap, RBAC Roles
[bootstrap-token] configured RBAC rules to allow Node Bootstrap tokens to get nodes
[bootstrap-token] configured RBAC rules to allow Node Bootstrap tokens to post CSRs in order for nodes to get long term certificate credentials
[bootstrap-token] configured RBAC rules to allow the csrapprover controller automatically approve CSRs from a Node Bootstrap Token
[bootstrap-token] configured RBAC rules to allow certificate rotation for all node client certificates in the cluster
[bootstrap-token] Creating the "cluster-info" ConfigMap in the "kube-public" namespace
[kubelet-finalize] Updating "/etc/kubernetes/kubelet.conf" to point to a rotatable kubelet client certificate and key
[addons] Applied essential addon: CoreDNS
[addons] Applied essential addon: kube-proxy

Your Kubernetes control-plane has initialized successfully!

To start using your cluster, you need to run the following as a regular user:

  mkdir -p $HOME/.kube
  sudo cp -i /etc/kubernetes/admin.conf $HOME/.kube/config
  sudo chown $(id -u):$(id -g) $HOME/.kube/config

Alternatively, if you are the root user, you can run:

  export KUBECONFIG=/etc/kubernetes/admin.conf

You should now deploy a pod network to the cluster.
Run "kubectl apply -f [podnetwork].yaml" with one of the options listed at:
  https://kubernetes.io/docs/concepts/cluster-administration/addons/

Then you can join any number of worker nodes by running the following on each as root:

kubeadm join 10.154.0.61:6443 --token i6skv0.j53bfnee6vqvsv10 \
	--discovery-token-ca-cert-hash sha256:24fb7caf305ca43c934021fa14cb3e06b60f7532c40eb4cc1447cc245ee705cc 
node/camila-ubuntu-2204-test-pr-weave-2 already uncordoned
Waiting for kubernetes api health to report ok
node/camila-ubuntu-2204-test-pr-weave-2 not labeled
✔ Kubernetes Master Initialized
Kubernetes control plane is running at https://10.154.0.61:6443
CoreDNS is running at https://10.154.0.61:6443/api/v1/namespaces/kube-system/services/kube-dns:dns/proxy

To further debug and diagnose cluster problems, use 'kubectl cluster-info dump'.
namespace/kurl created
✔ Cluster Initialized
service/registry created
secret/kurl-troubleshoot-spec created
customresourcedefinition.apiextensions.k8s.io/installers.cluster.kurl.sh created
installer.cluster.kurl.sh/latest created
configmap/kurl-last-config created
configmap/kurl-current-config created
⚙  Addon weave 2.8.1-20230417
serviceaccount/weave-net created
role.rbac.authorization.k8s.io/weave-net created
clusterrole.rbac.authorization.k8s.io/weave-net created
rolebinding.rbac.authorization.k8s.io/weave-net created
clusterrolebinding.rbac.authorization.k8s.io/weave-net created
secret/weave-passwd created
daemonset.apps/weave-net created
⚙  Checking cluster networking
Checking if weave-net binary can be found in the path /opt/cni/bin/
lrwxrwxrwx 1 root root       18 May 16 15:24 weave-net -> weave-plugin-2.8.1
Waiting up to 10 minutes for node to report Ready
service/kurlnet created
pod/kurlnet-server created
pod/kurlnet-client created
Waiting for kurlnet-client pod to start
warning: Immediate deletion does not wait for confirmation that the running resource has been terminated. The resource may continue to run on the cluster indefinitely.
pod "kurlnet-client" force deleted
pod "kurlnet-server" force deleted
service "kurlnet" deleted
configmap/kurl-current-config patched
⚙  Addon openebs 3.3.0
customresourcedefinition.apiextensions.k8s.io/blockdeviceclaims.openebs.io created
customresourcedefinition.apiextensions.k8s.io/blockdevices.openebs.io created
namespace/openebs created
serviceaccount/openebs created
clusterrole.rbac.authorization.k8s.io/openebs created
clusterrolebinding.rbac.authorization.k8s.io/openebs created
configmap/openebs-ndm-config created
deployment.apps/openebs-localpv-provisioner created
deployment.apps/openebs-ndm-operator created
daemonset.apps/openebs-ndm created
⚙  Waiting for OpenEBS CustomResourceDefinitions to be ready
✔ OpenEBS CustomResourceDefinitions are ready
OpenEBS LocalPV will be installed as the default storage class.
storageclass.storage.k8s.io/default created
configmap/kurl-current-config patched
⚙  Addon minio 2022-10-20T00-55-09Z
namespace/minio created
secret/minio-credentials created
service/ha-minio created
service/minio created
persistentvolumeclaim/minio-pv-claim created
deployment.apps/minio created
statefulset.apps/ha-minio created
awaiting minio deployment
awaiting minio readiness
awaiting minio endpoint
awaiting minio deployment
awaiting minio readiness
awaiting minio endpoint
configmap/kurl-current-config patched
⚙  Addon contour 1.23.0
namespace/projectcontour created
namespace/projectcontour configured
customresourcedefinition.apiextensions.k8s.io/contourconfigurations.projectcontour.io created
customresourcedefinition.apiextensions.k8s.io/contourdeployments.projectcontour.io created
customresourcedefinition.apiextensions.k8s.io/extensionservices.projectcontour.io created
customresourcedefinition.apiextensions.k8s.io/httpproxies.projectcontour.io created
customresourcedefinition.apiextensions.k8s.io/tlscertificatedelegations.projectcontour.io created
serviceaccount/contour created
serviceaccount/contour-certgen created
serviceaccount/envoy created
role.rbac.authorization.k8s.io/contour created
role.rbac.authorization.k8s.io/contour-certgen created
clusterrole.rbac.authorization.k8s.io/contour created
rolebinding.rbac.authorization.k8s.io/contour created
rolebinding.rbac.authorization.k8s.io/contour-rolebinding created
clusterrolebinding.rbac.authorization.k8s.io/contour created
configmap/contour created
service/contour created
service/envoy created
deployment.apps/contour created
daemonset.apps/envoy created
job.batch/contour-certgen-v1.23.0 created
configmap/kurl-current-config patched
⚙  Addon registry 2.8.1
object store bucket docker-registry created
Command "format-address" is deprecated, use 'kurl netutil format-ip-address' instead
configmap/registry-config created
configmap/registry-velero-config created
secret/registry-s3-secret created
secret/registry-session-secret created
service/registry configured
deployment.apps/registry created
secret/registry-htpasswd created
secret/registry-htpasswd patched
secret/registry-creds created
secret/registry-creds patched
/securiti_ai_tenant/kurl/addons/registry/2.8.1/tmp /securiti_ai_tenant/kurl /securiti_ai_tenant
.....+....+.....+......+...+.+......+...+..+..........+..+......+.+.........+......+.....+...+......+.......+........+....+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++*.......+.........+..+......+.........+.........+............+...................+.........+..+...+....+..+...+..........+...+............+...+..................+..+...+.+...+...+..+.+...........+....+.....+.......+...+...........+.+..+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++*.....+.+........+......+.............+..+....+.....+.......+..+.+.........+.....+......+.+.....+....+......+......+.....+.........+....+.....+......+....+...........+......+.............+..+....+...+..+....+.....+.+.....+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
.....+........+...+.+...+......+..+.......+..+.+.........+........+...+...+......+.+.........+.....+......+...+....+..+....+..+....+...+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++*.+......+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++*...+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
-----
Certificate request self-signature ok
subject=CN = registry.kurl.svc.cluster.local
secret/registry-pki created
/securiti_ai_tenant/kurl /securiti_ai_tenant
waiting for the registry to start
configmap/kurl-current-config patched
⚙  Addon prometheus 0.60.1-41.7.3
customresourcedefinition.apiextensions.k8s.io/alertmanagerconfigs.monitoring.coreos.com serverside-applied
customresourcedefinition.apiextensions.k8s.io/alertmanagers.monitoring.coreos.com serverside-applied
customresourcedefinition.apiextensions.k8s.io/podmonitors.monitoring.coreos.com serverside-applied
customresourcedefinition.apiextensions.k8s.io/probes.monitoring.coreos.com serverside-applied
customresourcedefinition.apiextensions.k8s.io/prometheuses.monitoring.coreos.com serverside-applied
customresourcedefinition.apiextensions.k8s.io/prometheusrules.monitoring.coreos.com serverside-applied
customresourcedefinition.apiextensions.k8s.io/servicemonitors.monitoring.coreos.com serverside-applied
customresourcedefinition.apiextensions.k8s.io/thanosrulers.monitoring.coreos.com serverside-applied
Error from server (NotFound): deployments.apps "kube-state-metrics" not found
Error from server (NotFound): daemonsets.apps "node-exporter" not found
Error from server (NotFound): deployments.apps "grafana" not found
Error from server (NotFound): deployments.apps "prometheus-adapter" not found
Error from server (NotFound): alertmanagers.monitoring.coreos.com "main" not found
Error from server (NotFound): services "kube-state-metrics" not found
Error from server (NotFound): services "prometheus-operator" not found
Error from server (NotFound): services "grafana" not found
Error from server (NotFound): services "alertmanager-main" not found
Error from server (NotFound): services "prometheus-k8s" not found
namespace/monitoring created
serviceaccount/grafana created
serviceaccount/k8s created
serviceaccount/kube-state-metrics created
serviceaccount/prometheus-adapter created
serviceaccount/prometheus-alertmanager created
serviceaccount/prometheus-node-exporter created
serviceaccount/prometheus-operator created
role.rbac.authorization.k8s.io/grafana created
clusterrole.rbac.authorization.k8s.io/grafana-clusterrole created
clusterrole.rbac.authorization.k8s.io/k8s created
clusterrole.rbac.authorization.k8s.io/kube-state-metrics created
clusterrole.rbac.authorization.k8s.io/prometheus-adapter-resource-reader created
clusterrole.rbac.authorization.k8s.io/prometheus-adapter-server-resources created
clusterrole.rbac.authorization.k8s.io/prometheus-operator created
rolebinding.rbac.authorization.k8s.io/prometheus-adapter-auth-reader created
rolebinding.rbac.authorization.k8s.io/grafana created
clusterrolebinding.rbac.authorization.k8s.io/grafana-clusterrolebinding created
clusterrolebinding.rbac.authorization.k8s.io/k8s created
clusterrolebinding.rbac.authorization.k8s.io/kube-state-metrics created
clusterrolebinding.rbac.authorization.k8s.io/prometheus-adapter-hpa-controller created
clusterrolebinding.rbac.authorization.k8s.io/prometheus-adapter-resource-reader created
clusterrolebinding.rbac.authorization.k8s.io/prometheus-adapter-system-auth-delegator created
clusterrolebinding.rbac.authorization.k8s.io/prometheus-operator created
configmap/grafana created
configmap/grafana-config-dashboards created
configmap/k8s created
configmap/prometheus-adapter created
configmap/prometheus-alertmanager-overview created
configmap/prometheus-apiserver created
configmap/prometheus-cluster-total created
configmap/prometheus-controller-manager created
configmap/prometheus-etcd created
configmap/prometheus-grafana-datasource created
configmap/prometheus-grafana-overview created
configmap/prometheus-k8s-coredns created
configmap/prometheus-k8s-resources-cluster created
configmap/prometheus-k8s-resources-namespace created
configmap/prometheus-k8s-resources-node created
configmap/prometheus-k8s-resources-pod created
configmap/prometheus-k8s-resources-workload created
configmap/prometheus-k8s-resources-workloads-namespace created
configmap/prometheus-kubelet created
configmap/prometheus-namespace-by-pod created
configmap/prometheus-namespace-by-workload created
configmap/prometheus-node-cluster-rsrc-use created
configmap/prometheus-node-rsrc-use created
configmap/prometheus-nodes created
configmap/prometheus-nodes-darwin created
configmap/prometheus-persistentvolumesusage created
configmap/prometheus-pod-total created
configmap/prometheus-proxy created
configmap/prometheus-scheduler created
configmap/prometheus-workload-total created
secret/alertmanager-prometheus-alertmanager created
secret/grafana-admin created
service/prometheus-coredns created
service/prometheus-kube-controller-manager created
service/prometheus-kube-etcd created
service/prometheus-kube-proxy created
service/prometheus-kube-scheduler created
service/grafana created
service/kube-state-metrics created
service/prometheus-adapter created
service/prometheus-alertmanager created
service/prometheus-k8s created
service/prometheus-node-exporter created
service/prometheus-operator created
deployment.apps/grafana created
deployment.apps/kube-state-metrics created
deployment.apps/prometheus-adapter created
deployment.apps/prometheus-operator created
apiservice.apiregistration.k8s.io/v1beta1.custom.metrics.k8s.io created
daemonset.apps/prometheus-node-exporter created
alertmanager.monitoring.coreos.com/prometheus-alertmanager created
prometheus.monitoring.coreos.com/k8s created
prometheusrule.monitoring.coreos.com/k8s created
prometheusrule.monitoring.coreos.com/k8s-operator created
prometheusrule.monitoring.coreos.com/prometheus-alertmanager.rules created
prometheusrule.monitoring.coreos.com/prometheus-config-reloaders created
prometheusrule.monitoring.coreos.com/prometheus-etcd created
prometheusrule.monitoring.coreos.com/prometheus-general.rules created
prometheusrule.monitoring.coreos.com/prometheus-k8s.rules created
prometheusrule.monitoring.coreos.com/prometheus-kube-apiserver-availability.rules created
prometheusrule.monitoring.coreos.com/prometheus-kube-apiserver-burnrate.rules created
prometheusrule.monitoring.coreos.com/prometheus-kube-apiserver-histogram.rules created
prometheusrule.monitoring.coreos.com/prometheus-kube-apiserver-slos created
prometheusrule.monitoring.coreos.com/prometheus-kube-prometheus-general.rules created
prometheusrule.monitoring.coreos.com/prometheus-kube-prometheus-node-recording.rules created
prometheusrule.monitoring.coreos.com/prometheus-kube-scheduler.rules created
prometheusrule.monitoring.coreos.com/prometheus-kube-state-metrics created
prometheusrule.monitoring.coreos.com/prometheus-kubelet.rules created
prometheusrule.monitoring.coreos.com/prometheus-kubernetes-apps created
prometheusrule.monitoring.coreos.com/prometheus-kubernetes-resources created
prometheusrule.monitoring.coreos.com/prometheus-kubernetes-storage created
prometheusrule.monitoring.coreos.com/prometheus-kubernetes-system created
prometheusrule.monitoring.coreos.com/prometheus-kubernetes-system-apiserver created
prometheusrule.monitoring.coreos.com/prometheus-kubernetes-system-controller-manager created
prometheusrule.monitoring.coreos.com/prometheus-kubernetes-system-kube-proxy created
prometheusrule.monitoring.coreos.com/prometheus-kubernetes-system-kubelet created
prometheusrule.monitoring.coreos.com/prometheus-kubernetes-system-scheduler created
prometheusrule.monitoring.coreos.com/prometheus-node-exporter created
prometheusrule.monitoring.coreos.com/prometheus-node-exporter.rules created
prometheusrule.monitoring.coreos.com/prometheus-node-network created
prometheusrule.monitoring.coreos.com/prometheus-node.rules created
servicemonitor.monitoring.coreos.com/grafana created
servicemonitor.monitoring.coreos.com/k8s created
servicemonitor.monitoring.coreos.com/kube-state-metrics created
servicemonitor.monitoring.coreos.com/prometheus-alertmanager created
servicemonitor.monitoring.coreos.com/prometheus-apiserver created
servicemonitor.monitoring.coreos.com/prometheus-coredns created
servicemonitor.monitoring.coreos.com/prometheus-kube-controller-manager created
servicemonitor.monitoring.coreos.com/prometheus-kube-etcd created
servicemonitor.monitoring.coreos.com/prometheus-kube-proxy created
servicemonitor.monitoring.coreos.com/prometheus-kube-scheduler created
servicemonitor.monitoring.coreos.com/prometheus-kubelet created
servicemonitor.monitoring.coreos.com/prometheus-node-exporter created
servicemonitor.monitoring.coreos.com/prometheus-operator created
configmap/kurl-current-config patched
⚙  Persisting the kurl installer spec
configmap/kurl-config created

No resources found

		Installation
		  Complete ✔


The UIs of Prometheus, Grafana and Alertmanager have been exposed on NodePorts 30900, 30902 and 30903 respectively.




To access the cluster with kubectl:

    bash -l
Kurl uses /etc/kubernetes/admin.conf, you might want to unset KUBECONFIG to use .kube/config:

    echo unset KUBECONFIG >> ~/.bash_profile



Node join commands expire after 24 hours.

To generate new node join commands, run cat tasks.sh | sudo bash -s join_token on this node.

To add worker nodes to this installation, run the following script on your other nodes:
    cat join.sh | sudo bash -s kubernetes-master-address=10.154.0.61:6443 kubeadm-token=i6skv0.j53bfnee6vqvsv10 kubeadm-token-ca-hash=sha256:24fb7caf305ca43c934021fa14cb3e06b60f7532c40eb4cc1447cc245ee705cc kubernetes-version=1.23.17 docker-registry-ip=10.96.2.26 primary-host=10.154.0.61
```